procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,87 @@
+- An estimated time span for the item
+- A short description of the *goal(s)* of the item
+- An optional proposal (something that requires review and approval through the [decision making process][decisions])
+  - *Note*: proposals require another Council member to second the proposal before it can be brought forward.
+
+The agenda is laid out in temporal order which is determined largely by the urgency of each topic (most urgent comes first). However, this is not a hard rule as, for example, it might make sense to keep one large discussion (even if it is the most urgent) for last in the meeting so that it can simply fill the entire remainder of the meeting time.
+
+[decisions]: http://example.com (**TODO**: link when decision making process is documented)
+
+### Meeting structure
+
+In addition to the main agenda items, every meeting has the following structure:
+
+- **Check-in**
+  - The check-in is a chance for Council members to connect on a human level and share how their doing outside of the context of the meeting. Examples of things shared in the check-in include: "I'm not feeling well because of my allergies, and so I'm not in a good headspace", "My partner and I just adopted a cat so I'm doing great!", and "I'm doing fine - nothing to report".
+- **Assigning roles**
+  - The Council members decide by consensus who will fill the roles of facilitator and note taker.
+- **Turn on recording**
+  - Recording is started
+- **Consent to the agenda**
+  - This is a chance for last minute changes to the agenda to be surfaced and to ensure everyone is on the same page as far as what will be discussed.
+- **Main agenda**
+  - The main topics of the meeting which have just been consented to
+- **Clarifications**
+  - A call is made for any last minute clarifications, questions, requests for additional action.
+- **Turn off recording**
+  - The recording is stopped
+- **Check-out**
+  - This is a chance for council members to express how they felt the meeting went including areas for improvement, particular highlights and kudos, or any other relevant meta aspects of the meeting.
+
+A template for this meeting structure that can be used to boostrap the agenda making process can be found [here](../templates/sync-meeting-agenda.md).
+
+## Meeting Decorum 
+
+The following are the social expectations that are expected of meeting participants:
+
+- **Usage of "hands"**: all participants are expected to use the "raise hands" feature of the video meeting software to put themselves into a queue to be called on by the facilitator. There are a few exceptions to this however: the facilitator when performing the duties of that role and the scribe when asking clarifying questions.
+- **Points of order**: every participant is encouraged to raise "points of order"  which are meta commentary on some aspect of the meeting that needs clarification or discussion.
```

Comment:
```suggestion
- **Points of order**: every participant is encouraged to raise "points of order"  which are meta commentary on some aspect of the meeting that needs clarification or discussion. If a point of order is considered urgent, it may be called out verbally - bypassing the usual queuing mechanism. Such interjections should be as brief as possible, to limit talking over one another. "Point of Order - we are out of our allotted time today", might be a suitable interjection.
```